fetch.sh

```diff
@@ -39,7 +45,10 @@ QCOW2="$(basename "$TMP_DIR"/*.ovf .ovf).qcow2"
 qemu-img convert -f vmdk -O qcow2 "$VMDK" "$QCOW2"
 
 # Remove now-useless files
-rm "$VMDK" "$TMP_DIR"/*.ovf
+rm "$VMDK" "$TMP_DIR"/*.ov[fa]
 
-echo Finished! Delete "$TMP_DIR" to tidy up
+echo Finished! Deleting "$TMP_DIR" to tidy up
+ls -la "$TMP_DIR"
+rm -fr "$TMP_DIR"
```

Comment:
I didn't do this on purpose. As hinted in the docs, I like to tweak the VM image a bit (get IE to launch on startup, set homepage), once I'm satsfied I've not done something stupid, then I remove the TMP_DIR. Or just get around to removing it when I'm short on space :)

Secondly, ``rm -rf`` isn't something anyone likes to see in scripts, and doesn't seem necessary off the top of my head.